src/components/schedule-page/new-month-page.component.tsx

```diff
@@ -11,30 +11,50 @@ import { Button } from "../common-components";
 import { useScheduleConverter } from "./import-buttons/hooks/use-schedule-converter";
 import { ScheduleKey } from "../../api/persistance-store.model";
 import { MonthSwitchActionCreator } from "../../state/reducers/month-state/schedule-data/month-switch.action-creator";
+import { LocalStorageProvider } from "../../api/local-storage-provider.model";
 
 export function NewMonthPlanComponent(): JSX.Element {
   const state = useSelector((state: ApplicationStateModel) => state);
   const currentMonth = state.actualState.persistentSchedule.present.schedule_info.month_number;
   const currentYear = state.actualState.persistentSchedule.present.schedule_info.year;
+  const { revision } = state.actualState;
```

Comment:
To jest dość istotna linjka. W stanie przechowyujemy teraz inforamcję o tym jaką rewizję wyświetlamy. Dałem to do globalnego stanu, bo potrzebujemy w różnych miejsach w apce, np. w zarządzaniu.